recipes-devtools/rust/rust_1.12.1.bb

```diff
@@ -0,0 +1,26 @@
+inherit rust-installer
+require rust.inc
+require rust-source-${PV}.inc
+
+DEPENDS += "cmake-native"
```

Comment:
Ah, a bug in the configure script then? (shouldn't check if it isn't going to be building llvm, and in our case it isn't).

Sounds fine, might be good to note that it is only included to workaround that.